src/universal/redux/segmentActions.js

```diff
@@ -1,22 +1,75 @@
 import {EventTypes} from 'redux-segment';
+import {cashay} from 'cashay';
+import {getAuthQueryString, getAuthedOptions} from './getAuthedUser';
 
 /**
  * Sometimes, we just want to track events in segment.
  */
 
 const SEGMENT_EVENT = '@@segment/EVENT';
 
-export function segmentEvent(event, properties) {
-  return {
-    type: SEGMENT_EVENT,
-    meta: {
-      analytics: {
-        eventType: EventTypes.track,
-        eventPayload: {
-          event,
-          properties
+const defaultProfile = {
+  avatar: null,
+  email: null,
+  id: null,
+  name: null
+};
+
+export function selectSegmentProfile(state) {
+  if (!state.auth || !cashay.store) { return defaultProfile; }
+  const userId = state.auth.obj.sub;
+  const {user} = cashay.query(getAuthQueryString, getAuthedOptions(userId)).data;
+  if (!user) { return defaultProfile; }
+
+  return ({
+    avatar: user.picture || null,
```

Comment:
since we request these from cashay, they'll always be guaranteed to be `null` until the response comes back, so we don't need a default value. Perhaps we want to send some kind of identifier though? An IP address?